gustaf/spline/extract.py

```diff
@@ -395,6 +395,26 @@ def control_mesh(spline,):
             "Supports 1 to 3."
         )
 
+def beziers(spline):
+  """
+  Extracts Bezier-type objects of any spline-type object
+
+  Parameters
+  ----------
+  spline : Gustaf-Spline
+    """
+  from gustaf.spline.base import Bezier, RationalBezier
+  if "Bezier" in spline.whatami:
+      return spline
+  elif "BSpline" in spline.whatami:
+      return [Bezier(**s.todict()) 
+          for s in super(type(spline), spline).extract_bezier_patches()]
+  elif "NURBS" in spline.whatami:
+      return [RationalBezier(**s.todict()) 
+          for s in super(type(spline), spline).extract_bezier_patches()]
```

Comment:
```suggestion
      return [
          RationalBezier(**s.todict()) 
          for s in super(type(spline), spline).extract_bezier_patches()
      ]
```
Generally, we should avoid multi-line list-comprehensions. But they are just too attractive I guess